UnitTests/Extensions/Test_ValueTypeExtensions.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.Extensions
+{
+    [TestClass]
+    public class Test_ValueTypeExtensions
+    {
+        [TestCategory("ValueTypeExtensions")]
+        [TestMethod]
+        public void Test_ValueTypeExtensions_ToHexString()
+        {
+            Assert.AreEqual(((byte)0).ToHexString(), "0x00");
```

Comment:
Yup, the one for `Color` is this one:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp/Helpers/ColorHelper.cs#L124-L132

I've used the `To____String` naming scheme as it's the same one used in the main `Microsoft.Toolkit` package (there's also [`ToArrayString`](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit/Extensions/ArrayExtensions.cs#L111-L120)), so I picked `ToHexString` for consistency within this package. In fact it's the same I did for the `Type` extension as well, since that's `ToTypeString`. Personally I like the `ToHexString` more, as I think it makes the intent clearer especially for less experienced developers. As in, it's returning a hexadecimal `string`, it's not "converting" the input value to hexadecimal somehow or making some kind of other transformation. Having the naming be closer to the classic `ToString` makes it immediate to understand: "ah, it's just like `ToString`, just with a different formatting".

What do you think? 😄

Reply:
Sounds good, maybe we can add a comment to the #3062 that we should rename the function in ColorHelper for consistency in the 7.0 release?

Reply:
Good idea! Done: https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3062#issuecomment-597874095 👍